changelog/unreleased/issue-3334

```diff
@@ -0,0 +1,7 @@
+Bugfix: `created new cache in' message is only printed on a terminal
+
+`created new cache` message was outputed even when the output wasn't a
+terminal. That broke piping `restic dump` output to tar or zip
+if cache directory didn't exist.
```

Comment:
Please add a (positive) final sentence which either states `This has been fixed` or something more specific like `The message is now only printed on a terminal`.